lib/portage/_global_updates.py

```diff
@@ -97,21 +97,17 @@ def _do_global_updates(trees, prev_mtimes, quiet=False, if_mtime_changed=True):
                     )
                     if not quiet:
                         writemsg_stdout(
-                            _(
-                                "  %s='update pass'  %s='binary update'  "
-                                "%s='/var/db update'  %s='/var/db move'\n"
-                                "  %s='/var/db SLOT move'  %s='binary move'  "
-                                "%s='binary SLOT move'\n  %s='update /etc/portage/package.*'\n"
-                            )
-                            % (
-                                bold("."),
-                                bold("*"),
-                                bold("#"),
-                                bold("@"),
-                                bold("s"),
-                                bold("%"),
-                                bold("S"),
-                                bold("p"),
+                            "  ".join(
+                                (
+                                    f"  {bold('.')}='update pass'",
```

Comment:
Just a trivial style preference: instead of including the leading space in the first string, drop it and instead add an empty string `""` as the first element of the tuple.